msg.go

```diff
@@ -403,35 +403,31 @@ Loop:
 	return string(s), off1, nil
 }
 
-func packTxt(txt []string, msg []byte, offset int, tmp []byte) (int, error) {
+func packTxt(txt []string, msg []byte, offset int) (int, error) {
 	var err error
+	if offset >= len(msg) {
+		return offset, ErrBuf
+	}
 	if len(txt) == 0 {
-		if offset >= len(msg) {
-			return offset, ErrBuf
-		}
 		msg[offset] = 0
 		return offset, nil
```

Comment:
[ Quoting <notifications@github.com> in "Re: [miekg/dns] Remove temporary bu..." ]
>>  	if len(txt) == 0 {
>> -		if offset >= len(msg) {
>> -			return offset, ErrBuf
>> -		}
>>  		msg[offset] = 0
>>  		return offset, nil
>
>should offset be incremented or is that happening somewhere else?

hmm, good question

/Miek

--
Miek Gieben